challenges/1-media-alunos/php/danesi/challenge.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+$primeiraNota = $argv[1];
+$segundaNota = $argv[2];
```

Comment:
Não seria 0 e 1 nos índices do argv?
OBS: não manjo de php, mas pelo oq vi na [doc](http://alvinalexander.com/php/php-read-command-line-arguments-in-php/) é isso.

Reply:
É 1 e 2 mesmo, o índice 0 é o nome do arquivo executado.
![image](https://user-images.githubusercontent.com/11967494/95041074-08665d80-06ac-11eb-8142-a55006101d38.png)